client/my-sites/backup/backup-contents-page/file-browser/file-type-icon.tsx

```diff
@@ -0,0 +1,20 @@
+import { Icon } from '@wordpress/components';
+import { file, pages } from '@wordpress/icons';
+import { FunctionComponent } from 'react';
+import { FileType } from './types';
+
+interface FileTypeIconProps {
+	type: FileType;
+}
+
+// @TODO: Add a filename prop and use it to determine the icon for files
```

Comment:
We might just want to set it somewhere more generally accessible, like in the utils.js transform, maybe just setting type there based on extension, hasChildren, etc.  Then when won't have to add extension checking again when we want to figure out if we can request previews since we'll be looking for image extensions, text extensions, etc.

Reply:
Just refactored this a bit and now we identify the file type in the utils.js transform and here we just decide which icon to display.